packages/firestore/src/local/memory_persistence.ts

```diff
@@ -184,7 +184,9 @@ export interface MemoryReferenceDelegate extends ReferenceDelegate {
 }
 
 export class MemoryEagerDelegate implements MemoryReferenceDelegate {
-  private inMemoryPins: ReferenceSet | null = null;
+  /** Manages all documents that are active in Query views. */
```

Comment:
Nit: I always find that `manage` is one of the words that can usually be replaced with something more specific. Even `keeps track of`, to me, seems more specific.

Reply:
I hope this is not a general stance on management :) Updated to use "track".